manifests/base/jupyter-rocm-pytorch-notebook-imagestream.yaml

```
@@ -16,7 +16,7 @@ spec:
   tags:
     # N Version of the image
     - annotations:
-        opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"},{"name":"ROCm-PyTorch","version":"2.3"}]'
+        opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.11"},{"name":"ROCm-PyTorch","version":"2.4.1"}]'
```

We use just major.minor:
```suggestion
        opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.11"},{"name":"ROCm-PyTorch","version":"2.4"}]'
```